script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -5,190 +5,190 @@ My body temperature drops.
 <Page19>
 The question from earlier of whether I could kill them or not comes to mind.
-Can I kill something that looks like a person, regardless of whether or not it is human?
+Can I kill something that looks like a person, regardless of whether or not it is really human?
 <Page20>
-―――It doesn't matter.//text overflow, needs rephrasing
+―――It doesn't matter.//Do not alter this page without playtesting
 My head simply hurts right now.
-This is the first time I've exposed my naked eyes for this long.
-This is the first time I've acted with such objective thought.
+The first time I've seen with my bare eyes for so long.//Crushed for page space
+The first time I've acted with such objective thought.//Crushed for page space
 As I look down on the lines of death in a frenzy,
-I store the images of the corpses I must kill into my memory.
+I memorize the images of the corpses I must kill.//Crushed for space
 Don't touch her. You're an eyesore. The dead should not eat the living.
 Besides―――
 The only creature that is allowed to prey on that woman is me.//ruby dots <そ|・><の|・><女|・><を|・><獲|・><物|・><に|・><す|・><る|・><な|・>
 <Page21>
 I cut the lines of the three corpses.
-Their torsos are split into three parts. Their bodies are cut open like butter. // 死亡する is dropped from the end here.
-It's that simple. Their attention was focused on Arcueid after all. How can you even be that defenseless?
+Each of them are split into three pieces. Like cutting through butter, the corpses are slit open and perish.//胴体をそれぞれ三分割。死体たちはバターを切るように割かれ、死亡する。
+Too easy. Their attention was focused on Arcueid after all. How can you even be that defenseless?
 <Page22>
 "Wake up! How long are you going to stay asleep for...!"
-There's no need to come in contact with Arcueid.
+There's no need to come in contact with Arcueid.//Is that accurate? Definitely not saying that none of the corpses made it so far as to touch Arc?  アルクェイドに触れるまでもない。
 I call out to her as I kill the fourth corpse.
 <Page23>
 "Mmm... Shi... ki?"
-Whether it was my call that brought her back to consciousness or not, Arcueid faintly opens her eyes.
+Perhaps regaining consciousness as a result of my call, Arcueid weakly opens her eyes.//俺の呼びかけで意識が戻ったのか、アルクェイドは弱々しく目を開ける。
 <Page24>
 "W-What are you doing here? No, run away...!"
 <Page25>
-As Arcueid begins to sit up, her face contorts as she curls her body.
+As Arcueid begins to sit up, her face contorts and she doubles over.//アルクェイドは体を起こしかけ、顔を歪めて体を丸めた。
 She's back to being conscious. Her life is still intact. But her wound is not healed.
 <Page26>
 She can't move from here yet.
-The one that should be running is her and not me. But unfortunately for her, I cannot allow that to happen.
+The one that should be running is her and not me. But unfortunately for her, I cannot allow that to happen.//Her can't afford it to happen in that he's relying on her still? Or just can't let her run in this state? 逃げるべきは俺ではなくコイツの方だが、あいにく、俺にはそれを許す余裕はない。
```

Comment:
He has no choice but to rely on her even though she should be resting.

Maybe 'But unfortunately for her, I don't have the leeway for that to happen.'